packages/botonic-plugin-flow-builder/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@botonic/plugin-flow-builder",
-  "version": "0.37.0",
+  "version": "0.37.1-alpha.1",
```

Comment:
Doubt: Why updating the version of the Flow Builder plugin?

Reply:
Because I had published an alpha and it was not in any PR. 
But when I test alphas I like that the bot has only one version of @botonic/react and the plugin-flow-builder uses @botonic/react internally.